_spikes-poc/function-fs-access/functions/long-runner/v1.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+let logger
+
+function delay (ms) {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, ms)
+  })
+}
+
+const v1 = {
+  process: async function (loggerArg) {
```

Comment:
@jon-shipley can you provide some details about the purpose of this function?
what is the benefit here? 🤔 
Are we just verifying that a function execution time can reach the 2hrs milestone?

Reply:
Yes, exactly that

On the Consumption plan functions time out after 10 minutes at the maximum.  On the App service plan functions can run for as long as you want.  This was just proving that.